apps/myaccount/src/components/create-password/create-password.tsx

```diff
@@ -52,7 +47,7 @@ export const CreatePassword: FunctionComponent<CreatePasswordPropsInterface> = (
     } = props;
 
     const { t } = useTranslation();
-    const config = useSelector((state: AppState) => state.config);
+    const config: any = useSelector((state: AppState) => state.config);
```

Comment:
Let's use a specific type here. You can update this to directly get the `productName` since that's the only usage of the `config` object here.

Reply:
Fixed with [082c7c8](https://github.com/wso2/identity-apps/pull/6518/commits/082c7c8b41afd8b7e4578895eb751cd7044a2f41)